src/main/java/hudson/plugins/report/jck/ReportParserCallable.java

```diff
@@ -67,12 +70,18 @@ public ReportParserCallable(String reportMatcherGlob, ReportParser reportParser)
                     .filter(e -> e != null)
                     .sorted()
                     .collect(Collectors.toList());
+            if(result.size() == 0){
+                result.add(getFakeSuite());
```

Comment:
Is it worthy to add (before this condition) also check for result == null?
```
 if(result == null){
        result=new ArrayList();
                result.add(getFakeSuite());
}
```

Reply:
I'm not sure and I don't see it in code that it could be null. Added for sure.

new ArrayList() ... leads to `java.io.IOException: Unable to serialize` later in results processing. I've played little bit around it and decided to use already initialized object and just add fake suite because it's sufficient.